Test needs update for non-acaimage version of DARK40

Oops, I forgot to come back around to fix this from https://github.com/sot/proseco/pull/292

```
======================================== FAILURES =========================================
_________________________________ test_imposters_on_guide _________________________________

    def test_imposters_on_guide():
        """Test the check for imposters by adding one imposter to a fake star"""
        stars = StarsTable.empty()
        # Add two stars because separate P2 tests seem to break with just one star
        stars.add_fake_constellation(n_stars=5, mag=9.5)
        mag = 8.0
        cnt = mag_to_count_rate(mag)
        stars.add_fake_star(id=110, row=100, col=-200, mag=mag)
        dark_with_badpix = DARK40.copy()
>       dark_with_badpix.aca[100, -200] = cnt * 0.1
E       AttributeError: 'numpy.ndarray' object has no attribute 'aca'

sparkles/tests/test_checks.py:227: AttributeError
========================== 1 failed, 16 passed in 79.81 seconds ==========================
```

--- a/sparkles/tests/test_checks.py
+++ b/sparkles/tests/test_checks.py
@@ -224,8 +224,8 @@ def test_imposters_on_guide():
     cnt = mag_to_count_rate(mag)
     stars.add_fake_star(id=110, row=100, col=-200, mag=mag)
     dark_with_badpix = DARK40.copy()
-    dark_with_badpix.aca[100, -200] = cnt * 0.1
-    dark_with_badpix.aca[100, -201] = cnt * 0.1
+    dark_with_badpix[100 + 512, -200 + 512] = cnt * 0.1
+    dark_with_badpix[100 + 512, -201 + 512] = cnt * 0.1
     aca = get_aca_catalog(**mod_std_info(n_fid=0, n_guide=8), stars=stars, dark=dark_with_badpix,
                           raise_exc=True)
     aca = ACAReviewTable(aca)

